src/components/index.js

```diff
@@ -0,0 +1,243 @@
+//                 <InputLabel id="demo-simple-select-outlined-label-4">Destination Country</InputLabel>
+//                 <Select
+//                   className={styles.text_field_styles}
+//                   name="toCountry"
+//                   label="Destination Country"
+//                   labelId="demo-simple-select-outlined-label-4"
+//                   value={state.toCountry}
+//                   onChange={onChange}>
+//                   <MenuItem value="country">Country</MenuItem>
+//                 </Select>
+//               </FormControl>
+
+//               <FormControl variant="outlined" className={styles.inputs}>
+//                 <InputLabel id="demo-simple-select-outlined-label-5">Destination City</InputLabel>
+//                 <Select
+//                   className={styles.text_field_styles}
+//                   name="toCity"
+//                   label="Destination City"
+//                   labelId="demo-simple-select-outlined-label-5"
+//                   value={state.toCity}
+//                   onChange={onChange}>
+//                   <MenuItem value="city">City</MenuItem>
+//                 </Select>
+//               </FormControl>
+
+
+//               <FormControl variant="outlined" className={styles.inputs}>
+//                 <InputLabel id="demo-simple-select-outlined-label-6">Accommodation</InputLabel>
+//                 <Select
+//                   className={styles.text_field_styles}
+//                   name="accommodation"
+//                   label="Accommodation"
+//                   labelId="demo-simple-select-outlined-label-6"
+//                   value={state.accommodation}
+//                   onChange={onChange}>
+//                   <MenuItem value="yes">Yes</MenuItem>
+//                   <MenuItem value="no">No</MenuItem>
+//                 </Select>
+//               </FormControl>
+//             </Grid>
+//           </Grid>
+//         </form>
+//       </Stepper>
+
+//       <div>
+//         {activeStep === steps.length ? 
+//         <Button 
+//         className={styles.submit_button}
+//         onClick={onSubmit}
+//         type="submit"
+//         color="primary"
+//         variant="contained"
+//         >
+//           Submit
+//         </Button> : (
+//             <div>
+//               <Typography className={classes.instructions}>{getStepContent(activeStep)}</Typography>
+//               <div className={styles.progress_buttons}>
+//                 <Button disabled={activeStep === 0} onClick={handleBack} className={classes.button}>
+//                   Back
+//               </Button>
+//                 <Button
+//                   variant="contained"
+//                   onClick={handleNext}
+//                   color="primary"
+//                   className={classes.button}
+//                 >
+//                   {activeStep === steps.length - 1 ? 'Finish' : 'Next'}
+//                 </Button>
```

Comment:
@Moise1 I realized you already have a component for new trip requests so I suggest you remove this one instead of commenting everything.